docs/VLSI/Tutorial.rst

```diff
@@ -141,9 +137,36 @@ To run DRC & LVS, and view the results in Calibre:
 
 .. code-block:: shell
 
-    make drc
+    make drc CONFIG=TinyRocketConfig
     ./build/drc-rundir/generated-scripts/view-drc
-    make lvs
+    make lvs CONFIG=TinyRocketConfig
     ./build/lvs-rundir/generated-scripts/view-lvs
 
 Some DRC errors are expected from this PDK, as explained in the `ASAP7 plugin readme <https://github.com/ucb-bar/hammer/tree/master/src/hammer-vlsi/technology/asap7>`__.
+Furthermore, the dummy SRAMs that are provided do not have any geometry inside, so will certainly cause DRC and LVS errors.
+
+Simulation
+^^^^^^^^^^
+Simulation with VCS is supported, and can be run at the RTL- or gate-level (post-synthesis and P&R). The simulation infrastructure as included here is intended for running RISC-V binaries on a Chipyard config. For example, for an RTL-level simulation:
+
+.. code-block:: shell
+
+    make sim-rtl CONFIG=TinyRocketConfig BINARY=$RISCV/riscv64-unknown-elf/share/riscv-tests/isa/rv64ui-p-simple
+
+Post-synthesis and post-P&R simulations use the ``sim-syn`` and ``sim-par`` targets, respectively.
+
+There are also ``-debug`` and ``-debug-timing``, which will instruct VCS to write a SAIF + VPD and do timing-annotated simulations, respectively. See the ``sim.mk`` file for all available targets.
+
+Note that for the ASAP7 example, gate-level simulations will timeout (the cause is being investigated).
```

Comment:
timeout in which tool?

Reply:
VCS. I will do my best to investigate by tutorial time.